### PR TITLE
Updated the Readme

### PR DESCRIPTION
**Pull request by hong3cooper**

### CODE REVIEW

**Review by evanrcooper — APPROVED** [review]
Looks good

**Review by malek1038 — APPROVED** [review]
LTGM!!

**Review by LaniW — COMMENTED** [review]
Wow, what an inspiring update!

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# github_review
\ No newline at end of file
+# github_review
+
+
```
Comment (lizocf) [inline README.md:3]: based

**Review by lizocf — COMMENTED** [review]
:D

**Review by secureshellprotocol — CHANGES_REQUESTED** [review]
add another newline between `github_review` and `It is 7:20PM, do you know where your children are?`

File: README.md (hunk 1 above)
Comment (Aidan4478D) [reply README.md:3]: looks good ☺

**Pull request merged**